packages/cosmos_ui_components/lib/cosmos_app_theme.dart

```diff
@@ -17,6 +17,11 @@ class CosmosAppTheme {
   static const radiusM = 8.0;
   static const radiusS = 4.0;
 
+  static const fontSizeXS = 12.0;
+  static const fontSizeS = 16.0;
+  static const fontSizeM = 20.0;
+  static const fontSizeL = 24.0;
```

Comment:
```suggestion
  static const fontSizeS = 12.0;
```
We're not using other sizes yet, so there is no point in adding those now, also I believe font size of 12 should be marked as `S`, since `XS` will fit better for the `10` size :)

Reply:
Awesome! ✅